src/encoded/static/components/item-pages/HiGlassViewConfigView.js

```diff
@@ -359,26 +361,32 @@ export class HiGlassViewConfigTabView extends React.PureComponent {
         return true;
     }
     /**
-     * checks whether multiple similar (within the same view and track, having same orientation and type) tilesets exist
+     * checks whether multiple similar (within the same view and track, having same type) tracks exist
      * @param {Object} updatedTilesetField edited field object
      * @param {Object} trackInfo track info object that holds detail view and track info
      */
     hasSimilarTilesets(updatedTilesetField, trackInfo) {
-        const hgc = this.getHiGlassComponent();
-        const currentViewConf = this.getHiGlassViewConfig(hgc);
-
-        if (!currentViewConf) {
-            throw new Error('Could not get current view configuration.');
-        }
-        const track = currentViewConf.views[trackInfo.vIndex].tracks[trackInfo.track];
-        const tileset = _.find(track, function (t) { return t.uid === trackInfo.uid; });
-
-        if (tileset) {
-            const { orientation, type } = tileset;
-            const similarTilesets = _.filter(track, (tilesetItem) => orientation === tilesetItem.orientation && type === tilesetItem.type);
-            return similarTilesets && similarTilesets.length > 1;
-        }
-        return false;
+      const hgc = this.getHiGlassComponent();
+      const currentViewConf = this.getHiGlassViewConfig(hgc);
+
+      if (!currentViewConf) {
+        throw new Error("Could not get current view configuration.");
+      }
+
+      // Gets all tracks with the same orientaton (e.g. all top tracks)
+      const tracks = currentViewConf.views[trackInfo.vIndex].tracks[trackInfo.track];
+      const currentTrack = _.find(tracks, function (t) {
+        return t.uid === trackInfo.uid;
```

Comment:
Codacy found an issue: [Expected indentation of 12 spaces but found 8.](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=6353740)